.kuzzlerc.sample

```diff
@@ -204,38 +204,71 @@
       "retryInterval": 1000
     },
 
-    // [cache]
+    // [internalCache]
     // The cache service relies on Redis
+    // sample settings for Redis service (see also https://github.com/luin/ioredis):
+    //
+    // 1. using a single Redis database:
+    //   node:
+    //     * host: The host on which Redis can be reached.
+    //       Can take an IP address, an URI or a hostname
+    //     * port: The port on which Redis is running
+    //   database: (optional) the ID of the redis database (default: 0)
+    "internalCache": {
+      "backend": "redis",
+      "node": {
+        "host": "redis",
```

Comment:
Isn't this configuration re-introducing the problem detailed in #400 (and solved in #406)?